schema/BUILD.bazel

```
@@ -63,6 +63,7 @@ go_library(
     srcs = [
         "bitbucket_server_util.go",
         "bitbucketcloud_util.go",
+        "codypro_util.go",
```

Very nit, how about we call this:
```suggestion
        "cody_pro_util.go",
```
Otherwise, the word "codypro" will trigger my spell checker each time, which would be slightly annoying.